### DISCUSSION TITLE
Where to place the Windows Binaries?

### DISCUSSION THREAD

**Discussion by ziggyedison** [apohl79/audiogridder#421] [Q&A]
Dear Andreas,

Thank you for creating this great project, can you please answer a few questions for me? 

I am running Windows 10 Pro and I have searched the forum, checked the Readme text and I may have missed it but could you please explain which folders to place the binaries in for the DAW Plugin on the first PC and the Server Binaries on the Second PC, also where does the config file go in each case?      (I thought there would be a small readme with the download files)

Thank you.

**Comment by apohl79** [comment]
Do the installers not work for you? That might be easier.

If they do not work, put the plugin dll's into your VST and VST3 folders accordingly.

The server exe runs from everywhere, but you can put it into the program files directory, if you like.

You don't need to create config files by hand. Just run the server and plugins.

**Reply by ziggyedison** [reply]
I have not tried the installers because I wanted to get a better understanding of the directory structuring for plugins generally,

**Comment by CaBuffalo** [comment]
Give AG a try. There is no harm to uninstall it if it doesn't work for you.

Most likely, it won't happen. Cheers.

On Sat, Mar 6, 2021 at 10:07 AM ziggyedison <notifications@github.com>
wrote:

> I have not tried the installers because I wanted to get a better
> understanding of the directory structuring for plugins generally, I am
> having other problems with plug-ins that do not conform to my DAW which is
> what led me to discover AG.
>
> I do not know if it was your intention, but one great use of your project
> maybe to overcome compatibility issues where plug-ins do not work in some
> setups and instances. Waves plug-ins are not working properly for me, and I
> run a less popular DAW (Motu's Digital Performer; DP) so Waves technical
> support basically refused to help or even answer my questions (because I am
> using DP in Windows and not macOS) even though some questions were generic
> and specifically about whether their plug-ins would function properly using
> an offload rack. Their Soundgrid offload App locked out my ASIO so I had to
> uninstall it.
>
> My logic was that if the plug-in will not correctly function in the DAW,
> if I could find a rack where their plug-ins did work, and that rack worked
> in my DAW I would have a working solution. Hopefully AG to the rescue.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/421#discussioncomment-436341>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EERNDASCZ662VXFDDTCJVPXANCNFSM4YW5E3PA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by johne53** [reply]
Can I ask a slightly dumb question..?

Let's say I've got 2 x Windows computers networked together and an installer to install some VST plugins. My DAW is on Machine-A so I guess I'd need to install the Audiogridder server onto Machine-B. But which machine do I need to install my plugins onto? Machine-A or Machine-B ??

[Edit...] FWIW I've installed about half a dozen VST3 plugins on both the server PC and also my local PC. In both cases, they're installed into:- **C:\Program Files\Common Files\VST3\Harrison\AVA**. I then added the  Audiogridder plugin to one of my DAW's channels (the DAW is Harrison Mixbus). But when I press the Audiogridder's '**+**' key, it doesn't list any installed plugins. Maybe Harrison products aren't supported yet?

BTW - in case it's relevant... both machines are running 64-bit Windows; the Harrison DAW is also 64-bit and I've installed 64-bit plugins.

**Comment by CaBuffalo** [comment]
If you want AG to host the plugin in order to save the CPU power of your
DAW, install the plugin on the server which is your Machine-B.

On Sun, Mar 21, 2021 at 4:11 AM johne53 ***@***.***> wrote:

> Can I ask a slightly dumb question..?
>
> Let's say I've got 2 x Windows computers networked together and an
> installer to install some VST plugins. My DAW is on Machine-A so I guess
> I'd need to install the Audiogridder server onto Machine-B. But which
> machine do I need to install my plugins onto? Machine-A or Machine-B ??
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/421#discussioncomment-509463>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DJB3IYACZB56ZRBALTEXH5BANCNFSM4YW5E3PA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by johne53** [reply]
Many thanks **CaBuffalo** - the plugin installer placed its plugins here:-

- **C:\Program Files\Common Files\VST3\Harrison\AVA**

Do they need to be in a specific folder - e.g.

- C:\Program Files\Common Files\VST3

Or if not, how will the server know where to look for them?

**Comment by CaBuffalo** [comment]
No, AG will scan the VST folders. If not, you can set the scan directory in
the setting.

On Sun, Mar 21, 2021 at 11:06 AM johne53 ***@***.***> wrote:

> Many thanks *CaBuffalo* - the plugin installer placed its plugins here:-
>
>    - *C:\Program Files\Common Files\VST3\Harrison\AVA*
>
> whereas AG's server has installed itself here:-
>
>    - C:\Program Files\Common Files\VST3
>
> Do they in fact need to be in the same folder?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/421#discussioncomment-510332>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6E3NDNVQUHRH5Z25PLTEYYSRANCNFSM4YW5E3PA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by johne53** [reply]
Thanks again for the quick reply.... I made a screenshot for you but I can't find any way to attach an image here :-(

Basically, the server installer seems to have installed just 3 files called, **AudioGridderServer.exe**, **unins000.exe** and **unins000.dat** - I assume the "unins" ones are an uninstaller?? So if I start **AudioGridderServer.exe** it launches the server but I can't find any way of seeing or changing any settings. Is there something extra I should have installed?

**Reply by apohl79** [reply]
The server adds a system tray icon. I suggest you watch the demo video on the website to get a little intro to how it works and you you use it.

**Reply by johne53** [reply]
Many thanks guys. I've watched the demo video twice but somehow, I'd missed the bit about the tray icon. An added problem is that for testing, I'd chosen a suite of VST3 plugins from a company called Harrison. Unfortunately, those ones don't seem to get recognised by Audiogridder. I've now swapped to some VST3's from an outfit called Blue Cat and those ones seem to be getting found.

The problem here now is that when I try to display any of the Blue Cat plugins, I see a very generic looking display (i.e. text based) rather than the graphical knobs / buttons / meters etc that I'd usually see with those plugins. But I'm guessing that migth be a problem with the DAW rather than with Audiogridder?

![Capture-02](https://user-images.githubusercontent.com/4126346/111963076-06be1400-8aeb-11eb-87e3-8157fc5ea532.png)

**Reply by apohl79** [reply]
You have enabled the "generic editor" in AG. Click the settings icon and disable it to see the plugin GUI.

**Reply by johne53** [reply]
Sorry to keep asking dumb questions but whereabouts is the Settings icon? Is it on the plugin's side somewhere? Here's what I see if I open my server settings::-

![Capture-03](https://user-images.githubusercontent.com/4126346/111984927-e64e8380-8b03-11eb-8d6e-56706c95f5a2.PNG)

**Reply by apohl79** [reply]
The plugin settings, not the server. Click the settings icon on the plugin. See (2) here: https://audiogridder.com/plugin/#plugin-window

**Reply by johne53** [reply]
Found it, and it all seems to be working fine now... great support here - many thanks!
